pkg/cmd/target.go

```diff
@@ -676,7 +683,13 @@ func getSeedForProject(shootName string) (seedName string) {
 	gardenClientset, err := gardencoreclientset.NewForConfig(NewConfigFromBytes(*kubeconfig))
 	checkError(err)
 	shootList, err := gardenClientset.CoreV1beta1().Shoots("").List(metav1.ListOptions{})
-	checkError(err)
+	// temporary solution , will clear up code in ticket move get seed  out of targetShoot method
```

Comment:
```suggestion
	// temporary solution , will clean up code in ticket move get seed out of targetShoot method #269
```
added reference to the issue